sdk/src/main/java/io/matchmore/sdk/monitoring/MatchMonitor.kt

```diff
@@ -0,0 +1,51 @@
+package io.matchmore.sdk.monitoring
+
+import io.matchmore.sdk.AlpsManager
+import io.matchmore.sdk.api.async
+import io.matchmore.sdk.api.models.Device
+import io.matchmore.sdk.api.models.Match
+import java.util.*
+
+interface MatchMonitorDelegate {
+    fun didReceiveMatches(matches: Set<Match>, forDevice: Device)
+}
+
+class MatchMonitor(private val manager: AlpsManager, private val delegate: MatchMonitorDelegate) {
+    var monitoredDevices = mutableSetOf<Device>()
+    var deliveredMatches = mutableSetOf<Match>()
+
+    private var timer: Timer? = null
+
+    fun startPollingMatchesA() {
+        val period: Long = 5000 // repeat every 10 sec.
+        timer = Timer()
+        timer?.scheduleAtFixedRate(object : TimerTask() {
+            override fun run() {
+                getMatches()
+            }
+        }, 0, period)
+    }
+
+    fun stopPollingMatches() {
+        timer?.cancel()
+        timer = null
+    }
+
+    private fun getMatches() {
```

Comment:
You can write it in one line if you want:
`private fun getMatches() = monitoredDevices.forEach { getMatchesForDevice(it) }`